### PR TITLE
framework/st_things : Modify logic when 'remove_query_parameter' occu…

### PR DESCRIPTION
**Pull request by YiHyunJin**
…red.

1. Currently, the query removes if and rt in any state.
2. However, if the query is 'if', 'if' is delete, and if it is 'rt', 'rt' is delete.

### CODE REVIEW

File: framework/src/st_things/st_things_request_handler.c [hunk 1]
```diff
@@ -831,8 +831,12 @@ bool handle_get_req_helper(const char *res_uri, const char *query, OCRepPayload
 	}
 	// Remove 'rt' and 'if' from query parameters
 	if (NULL != query && strlen(query) > 0) {
-		remove_query_parameter(req_msg->query, OC_RSRVD_RESOURCE_TYPE);
-		remove_query_parameter(req_msg->query, OC_RSRVD_INTERFACE);
+		if(strstr(req_msg->query, OC_RSRVD_RESOURCE_TYPE) != NULL) {
```
Comment (JeongJunSik) [inline framework/src/st_things/st_things_request_handler.c]: please space between if and (

**Comment by seinfra** [comment]
Target : [1f76e87fc744f3607c427a905452a0e9ab212e16] -

**Comment by seinfra** [comment]
Target : [1f76e87fc744f3607c427a905452a0e9ab212e16] -

**Comment by seinfra** [comment]
Target : [ec466d96e3cda44792ac11160966095a7a1f9731] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [ec466d96e3cda44792ac11160966095a7a1f9731] - Code Rule Check OK.

**Pull request merged**